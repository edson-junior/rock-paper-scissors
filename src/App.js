import React, { Component } from 'react';
import { render } from 'react-dom';

class App extends Component {
  render() {
    return (
      <div>
        <h3>Rock Paper Scissors</h3>
      </div>
    ) 
  }
}

export default App;
