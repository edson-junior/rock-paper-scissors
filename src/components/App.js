import React, { Fragment } from 'react';
import Play from './Play';

const App = () => (
  <Fragment>
    <img className="logo" src="/Logotype_horizontal.svg" width="300" height="77" alt="" />
    <hr />
    <Play />
  </Fragment>
);

export default App;
