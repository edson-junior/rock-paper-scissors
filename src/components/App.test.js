import React from 'react';
import App from './App';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

const app = shallow(<App />);

describe('App', () => {
  it('renders properly', () => {
    expect(toJson(app)).toMatchSnapshot();
  });

  it('should have a logo', () => {
    expect(app.find('img.logo').exists()).toBe(true);
  });

  it('should have a `Play` component', () => {
    expect(app.find('Connect(Play)').exists()).toBe(true);
  });
});
