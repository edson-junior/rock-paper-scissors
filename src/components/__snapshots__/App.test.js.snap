// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<React.Fragment>
  <img
    alt=""
    className="logo"
    height="77"
    src="/Logotype_horizontal.svg"
    width="300"
  />
  <hr />
  <Connect(Play) />
</React.Fragment>
`;
